### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by lenaorobei**
## Description

Added Node.js 18

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "main": "src/index.js",
   "description": "Extensibility template for AEM Content Fragment Admin Console",
   "engines": {
-    "node": "^14 || ^16"
+    "node": "^14 || ^16 || ^18"
   },
```
Comment (irenelagno) [inline package.json:8]: I'd recommend to use only one supported engine similar to aio-cli:
 https://github.com/adobe/aio-cli/blob/master/package.json
and bump plugin version as well with this change

Comment (lenaorobei) [reply package.json:8]: Just to be safe, I would keep it. So that we are compatible with the previous versions.

**Review by slitviachenko — APPROVED** [review]

**Review by irenelagno — APPROVED** [review]

**Comment by purplecabbage** [comment]
This needs to be >=18 or ^18 || >= 20
https://github.com/adobe/aio-cli-plugin-app/blob/master/package.json#L77
Please update and publish as this is breaking app init in some cases.

**Comment by lenaorobei** [comment]
@purplecabbage the intention behind keeping `^14 || ^16` was to maintain compatibility with older versions of aio cli.

**Pull request closed without merging**